README.md

```diff
@@ -34,4 +34,28 @@ Here goes the control loop code
 # NODE_CONTROL_LOOP_ENDS
 ```
 If its not possible to find either `# HEADER_BEGIN` or `# HEADER_END`, the script will not be
-divided into two scripts and it will not be possible to have multiple URCap nodes in one program.
\ No newline at end of file
+divided into two scripts and it will not be possible to have multiple URCap nodes in one program.
+
+## Acknowledgment
+Developed in collaboration between:
+[<img height="60" alt="Universal Robots A/S" src="doc/resources/ur_logo.jpg">](https://www.universal-robots.com/) &nbsp; and &nbsp;
```

Comment:
If newline in the ROS driver, then also here!

Reply:
Resolved in 51da00a